### DISCUSSION TITLE
Proxmox Kernel Clean not work

### DISCUSSION THREAD

**Discussion by rollingafull** [tteck/Proxmox#943] [Help Needed] [answered]
![imagen](https://user-images.githubusercontent.com/62690523/212627311-92e41ea7-0519-49e3-ae2a-84f9ce099204.png)

For me this script is not working. It finish too fast and if i retry it show same kernels for clean.

Regards.

**Comment by tteck** [comment, marked as answer]
As `5.15` is the current default kernel in PVE 7.3 the package management directly depends on it, so it isn't possible while running `6.1` kernels. (the script tries to remove **ALL** old kernels)
When Proxmox switches to `6.1` as the new default, then it'll be possible. 

You can manually remove all but `pve-kernel-5.15.83-1-pve` (need at least 1 `5.15` kernel to keep `pve-headers-5.15`)

`apt purge pve-kernel-5.15.30-2-pve pve-kernel-5.15.60-1-pve pve-kernel-5.15.60-2-pve`

**Reply by rollingafull** [reply]
Yes i do it manualy.

Thanks for quick reply.

Regards.

**Reply by tteck** [reply]
I added a `pve-kernel-6.1` check to the script
